src/chart/bar/BarChart.tsx

```diff
@@ -134,12 +128,47 @@ const NeoBarChart = (props: ChartProps) => {
     return chartColorsByScheme[colorIndex];
   };
 
-  const BarComponent = ({ bar, borderColor }) => {
+  // Function to call from BarComponent. Conducts necessary logic for Report Action.
+  const handleBarClick = (bar, event) => {
```

Comment:
this function is unused, we should remove it

Reply:
Removed